### ISSUE TITLE
Improved Google maps integration

### ISSUE THREAD

**Issue by dave0** [dave0/leaguerunner#31]
Originally filed by trawna.com on 2007-02-03T02:10:35

There should be a way for Google maps to be dynamically generated,
including showing field orientation.  In 2006 TUC did this with hard-coded
values and a clunky JavaScript interface for maintenance.  There is clearly
a better way to do it, storing the required data in the field database, but
it will also take some clever JavaScript, interfacing with the Google maps
API, to provide a user-friendly administrative interface for maintaining
this data.

**Comment by dave0** [comment]
Updated by trawna.com on 2009-05-07T21:48:08

Added JavaScript field layout editor and viewer. The viewer is mostly complete and
linked from the field view page (through the "view map" link). The editor is complete
enough to be 95% useful, accessed through a new "edit layout" menu option. This
mostly obsoletes the location_url and layout_url columns, but they are left there for
the transition period, and used whenever a new-style field layout has not yet been drawn.

Future enhancements for the editor will be to handle parking locations (already
handled in the viewer, if you manually insert values into the database in
lat,lnt/lat,lng/lat,lng format) and ways to work with multiple fields at the same
site (e.g. aligning two fields, editing multiple fields at once).

Original ticket set status to Started (we converted to open)
